src/Export/Export.php

```diff
@@ -58,6 +58,12 @@ class Export
 	protected $confirmDialog = null;
 
 
+	/**
+	 * @var string
+	 */
+	protected $target = null;
```

Comment:
Could you please change that to `@var string|null`? Thx

Reply:
My bad, sorry. It is fixed now.